scico/linop/xray/astra.py

```diff
@@ -95,7 +100,10 @@ def project_world_coordinates(
     """Project world coordinates along ray into the specified basis.
 
     Project world coordinates along `ray` into the basis described by `u`
-    and `v` with center `d`.
+    and `v` with center `d`. The term ""world"" emphasizes that the
```

Comment:
Why double quotes on either side of "world"?

Reply:
I want that to print as 
```
The term "world" emphasizes...
```
using the quotes to indicate I'm talking about a word as word. I think the double double quotes escapes the double quotes to achieve that.

Reply:
I couldn't confirm this since the function in question isn't included in the rendered API docs, but it would be worth checking that this syntax usage is correct in case this function ever does get included in the docs.